### DISCUSSION TITLE
How can i use mapTypeControlOptions in @react-google-maps/api?

### DISCUSSION THREAD

**Discussion by cacastego** [JustFly1984/react-google-maps-api#1987] [Q&A]
I tried to use 
mapTypeControlOptions: {
    style: **google**.map.MapTypeControlStyle.DROPDOWN_MENU(),
},
But i get **'google'** is not defined” error

I need help

**Comment by JustFly1984** [comment]
there is a lot of issues closed around google is undefined error in this repository. please look better, it was solved a lot of times.
